scenario,status,details,timestamp
basic_data_to_server_setup,PASS,"{'mqtt_broker_enabled': True, 'mqtt_port_correct': True, 'mqtt_anonymous_enabled': True, 'mqtt_process_running': True, 'dts_enabled': True, 'period_correct': True, 'mqtt_server_correct': True, 'mqtt_topic_correct': True, 'mqtt_client_id_correct': True, 'mqtt_qos_correct': True, 'no_errors_in_logs': False, 'mqtt_message_received': True, 'raw_results': {'mqtt_enabled': ""mosquitto.mqtt.enabled='1'"", 'mqtt_port': ""mosquitto.mqtt.local_port='1883'"", 'mqtt_anonymous': ""mosquitto.mqtt.anonymous_access='1'"", 'mqtt_process': '6846 root      4708 S    /usr/sbin/mosquitto -c /tmp/mosquitto.conf', 'dts_instance': ""data_sender.1.name='NewInstance2'"", 'dts_enabled': ""data_sender.1.enabled='1'"", 'dts_period': ""data_sender.1.period='3'"", 'mqtt_server': ""data_sender.2.mqtt_host='test.mosquitto.org'"", 'mqtt_topic': ""data_sender.2.mqtt_topic='testing'"", 'mqtt_client_id': ""data_sender.2.mqtt_client_id='1'"", 'mqtt_qos': ""data_sender.2.mqtt_qos='1'"", 'log_errors': '625 Mon Feb 10 08:22:47 2025 daemon.err rms_mqtt[8293]: Failed to connect with authentication keys\n635 Mon Feb 10 08:33:02 2025 daemon.err rms_mqtt[8627]: Failed to connect with authentication keys\n640 Mon Feb 10 08:43:15 2025 daemon.err rms_mqtt[8957]: Failed to connect with authentication keys\n646 Mon Feb 10 08:54:10 2025 daemon.err rms_mqtt[9311]: Failed to connect with authentication keys\n652 Mon Feb 10 09:04:22 2025 daemon.err rms_mqtt[9641]: Failed to connect with authentication keys\n659 Mon Feb 10 09:14:31 2025 daemon.err rms_mqtt[9971]: Failed to connect with authentication keys\n671 Mon Feb 10 09:24:42 2025 daemon.err rms_mqtt[10537]: Failed to connect with authentication keys', 'mqtt_message': True}}",2025-02-10 09:29:23
